<launch>
   
   <!-- Node to record differential GPS points in a text file -->
   <node pkg="cutter_localization" type="gps_survey.py" name="gps_surveyor" output="screen"> 
      <param name="topic_in" value="/gps_pose" type="string"/>
      <param name="topic_out" value="/cwru/gps_pose_snowplow" type="string"/>
      <param name="filename" value="yomama.isfat" type="string"/>
   </node>
   
</launch>
